chain/network/src/peer/peer_actor.rs

```diff
@@ -547,15 +547,15 @@ impl PeerActor {
                 }
                 let genesis_id = self.network_state.genesis_id.clone();
                 if handshake.sender_chain_info.genesis_id != genesis_id {
-                    tracing::debug!(target: "network", "Received connection from node with different genesis.");
+                    tracing::debug!(target: "network", "received connection from node with different genesis");
                     self.send_message(&PeerMessage::HandshakeFailure(
                         self.my_node_info.clone(),
                         HandshakeFailureReason::GenesisMismatch(genesis_id),
                     ));
                     return;
                 }
                 if handshake.target_peer_id != self.my_node_info.id {
-                    tracing::debug!(target: "network", "Received handshake from {:?} to {:?} but I am {:?}", handshake.sender_peer_id, handshake.target_peer_id, self.my_node_info.id);
+                    tracing::debug!(target: "network", sender_peer_id = ?handshake.sender_peer_id, target_peer_id = ?handshake.target_peer_id, my_node_id = ?self.my_node_info.id, "received handshake to wrong target");
```

Comment:
This does not sound accurate to me. The previous message was much clearer. Maybe something like this:

```suggestion
                    tracing::debug!(target: "network", peer_id = ?handshake.sender_peer_id, intended_handshake_with = ?handshake.target_peer_id, my_node_id = ?self.my_node_info.id, "peer handshaking with a wrong node, disconnecting peer");
```